### DISCUSSION TITLE
Appreciation

### DISCUSSION THREAD

**Discussion by predaytor** [joe-bell/cva#137] [Show and tell]
I would like to thank the creator @joe-bell and the participants of this project. I put a star back when the library first came out, while working on a project with Stitches, researching the best way to style websites and creating a design system. Switching back to the vanilla css (which I overwhelmingly love) with css-modules, I couldn't figure out the best way to orchestrate the codebase until I remembered that this library exists! Such an incredible and simple API! Thank you!

**Comment by joe-bell** [comment]
Thank you!
